README.md

```diff
@@ -2,11 +2,11 @@
 
 ## Table of Contents
   - [Check-ins](#check-ins)
-  - [Approach To Code Design](#plan-for-project-organization)
-  - [Organization](#approaches-to-project-organization)
+  - [Organization](#plan-for-project-organization)
```

Comment:
Our approaches to organization section still isn't accounted for- unless we wanted to group it up with the plan?
In which case, can we change `Organization` to `Plan and Approaches To Organization`?

Reply:
Or change the headline of `Approaches to Project Organization` to a subheading?